### ISSUE TITLE
Refactor Data model

### ISSUE THREAD

**Issue by Joltrix** [StarcounterApps/Timeline#44]
The timeline should stop using the Simplified data model, and instead the database classes should exist locally.

**Comment by Mihaiii** [comment]
Is this a standard for all the apps?

**Comment by Joltrix** [comment]
Yes, the goal is to move away from the Simplified data model, @bouguila has already started with the people app for example.

So at first we will create new branches, where we move away from the simplified data model, and then "slowly" transition over to that version,

That's how I've understood it at least

**Comment by Mihaiii** [comment]
Thanks for the info.
This is a pretty big decision. Is there an internal discussion on github? If not, I really think we should have one, to see from other teams what could go wrong (e.g. are we sure it won't affect blending?).

cc @LarsTjernstrom
